install_test.sh

```diff
@@ -0,0 +1,11 @@
+if [[ $TRAVIS_OS_NAME == "linux" ]]; then 
```

Comment:
Add `set -e`

Reply:
just switched to `set -eo pipefail` to make sure we capture pipeline errors